### PR TITLE
Docs pass for v2.eventarc

### PR DESCRIPTION
**Pull request by joehan**
### Description
Docs review for v2 event arc - this one was looking pretty good but just needed some empty descriptions added.

### CODE REVIEW

**Review by pavelgj — APPROVED** [review]

File: src/v2/providers/eventarc.ts [hunk 1]
```diff
@@ -54,14 +54,26 @@ export interface EventarcTriggerOptions extends options.EventHandlerOptions {
   filters?: Record<string, string>;
 }
 
+/**
+ * A function that takes a CloudEvent.
```
Comment (egilmorez) [inline src/v2/providers/eventarc.ts:58]: Is CloudEvent a literal?

File: src/v2/providers/eventarc.ts [hunk 2]
```diff
@@ -54,14 +54,26 @@ export interface EventarcTriggerOptions extends options.EventHandlerOptions {
...
+ * @param handler - A function to execute when triggered.
+ * @returns A Cloud Function that you can export and deploy.
+ */
 export function onCustomEventPublished<T = any>(
   eventType: string,
   handler: CloudEventHandler
 ): CloudFunction<CloudEvent<T>>;
 
+/** Handles an Eventarc event.
+ * @param opts - Options to set on this Cloud Function
```
Comment (egilmorez) [inline src/v2/providers/eventarc.ts:73]: I strongly prefer just "function," and think "Cloud Functions" -- the capped product -- should always be plural.

**Review by egilmorez — COMMENTED** [review]
Couple nits, thanks Joe!

**Review by egilmorez — APPROVED** [review]
Thanks, LG!

**Pull request merged**